bootstrap.ps1

```diff
@@ -119,6 +122,7 @@ Param(
     [string]$Prefix,
     [string]$Dependency,
     [string]$Linkage = "shared",
+    [switch]$RemoveDeprecations,
```

Comment:
This variable is already being used below in line 341, but was not properly getting a value.